source/scss/_library/_objects.sections.scss

```diff
@@ -114,33 +114,48 @@
   }
 }
 
-.c-home {
-  &__sections {
-    padding-top: $space-double;
+// Homepage structure:
+//
+// banners (breaking news, product marketing)
+// content-top
+// tout (donation ask)
+// content-bottom
 
-    .o-box-banner + .c-featured-blocks {
-      margin-top: 0;
-    }
+.c-home__banners {
+  padding-top: $space-double;
 
-    .o-box-banner {
-      margin-bottom: 50px;
-    }
+  .l-container:only-child {
```

Comment:
Is there any other class you can hook into, or use an existing spacing utility class? I'd prefer to stay away from adding styles to nested layout classes and its reliance on it being the only child. It just feels a little brittle.

Reply:
i'll add a class. Also, going to change from `:only-child` to `:last-child`, since that's more accurately what i'm going for here.  Add a bottom-margin if this is at the bottom.